google-api/google_search.py

```diff
@@ -556,92 +559,8 @@ def daterange(start_date, end_date):
-        country           VARCHAR(255)    ENCODE ZSTD,
-        device            VARCHAR(255)    ENCODE ZSTD,
-        page              VARCHAR(2047)   ENCODE ZSTD,
-        position          FLOAT           ENCODE ZSTD,
-        clicks            DECIMAL         ENCODE ZSTD,
-        ctr               FLOAT           ENCODE ZSTD,
-        impressions       DECIMAL         ENCODE ZSTD,
-        node_id           VARCHAR(255)    ENCODE ZSTD,
-        page_urlhost      VARCHAR(255)    ENCODE ZSTD,
-        title             VARCHAR(2047)   ENCODE ZSTD,
-        theme_id          VARCHAR(255)    ENCODE ZSTD,
-        subtheme_id       VARCHAR(255)    ENCODE ZSTD,
-        topic_id          VARCHAR(255)    ENCODE ZSTD,
-        subtopic_id       VARCHAR(255)    ENCODE ZSTD,
-        subsubtopic_id    VARCHAR(255)    ENCODE ZSTD,
-        theme             VARCHAR(2047)   ENCODE ZSTD,
-        subtheme          VARCHAR(2047)   ENCODE ZSTD,
-        topic             VARCHAR(2047)   ENCODE ZSTD,
-        subtopic          VARCHAR(2047)   ENCODE ZSTD,
-        subsubtopic       VARCHAR(2047)   ENCODE ZSTD)
-        COMPOUND SORTKEY (date,page_urlhost,theme,page,clicks);
-
-ALTER TABLE cmslite.google_pdt OWNER TO microservice;
-GRANT SELECT ON cmslite.google_pdt TO looker;
-
-INSERT INTO cmslite.google_pdt
-SELECT gs.*,
-       COALESCE(themes.node_id, '') AS node_id,
-       SPLIT_PART(gs.page, '/', 3)  AS page_urlhost,
-       title,
-       theme_id,
-       subtheme_id,
-       topic_id,
-       subtopic_id,
-       subsubtopic_id,
-       theme,
-       subtheme,
-       topic,
-       subtopic,
-       subsubtopic
-FROM   google.googlesearch AS gs
-       LEFT JOIN google.google_sites r
-              ON gs.site = r.ref_site
-       -- fix for misreporting of redirected front page URL in Google search
-       LEFT JOIN cmslite.themes AS themes
-              ON CASE
-                   WHEN page = 'https://www2.gov.bc.ca/' THEN
-                   'https://www2.gov.bc.ca/gov/content/home'
-                   ELSE page
-                 END = themes.hr_url
-WHERE
-        gs.site NOT IN (
-            'sc-domain:gov.bc.ca',
-            'sc-domain:engage.gov.bc.ca'
-        )
-    -- Case where data collected by site and sc-domain overlaps
-        OR (
-            gs.site = 'sc-domain:gov.bc.ca'
-            AND page_urlhost = r.sc_urlhost
-            AND gs.DATE :: DATE < r.start_date :: DATE
-        )
-    -- All other sc-domain data, excluding sites collected directly
-        OR (
-            gs.site = 'sc-domain:gov.bc.ca'
-            AND r.sc_domain = 'f'
-            AND page_urlhost NOT IN (
-            SELECT
-                sc_urlhost
-            FROM
-                google.google_sites
-            WHERE
-                sc_urlhost IS NOT NULL
-            )
-    )
-    OR (gs.site = 'sc-domain:engage.gov.bc.ca');
-
-COMMIT; 
-"""
+{pdt_query}
+""".format(pdt_query=pdt_query)
```

Comment:
This seems redundant. Can you not just assign `pdt_query` to `query` without formatting? All this accomplishes is wrapping it in `\n` characters.

Maybe even simpler, could you not move the file open read down here? As in:
```python
# get sql for pdt build
query = open('dml/{}'.format(dml_file), 'r').read()
```

Then you can remove lines 242 and 243 while also simplifying this.